### ISSUE TITLE
Support loading Technical Grimoire background format instead

### ISSUE THREAD

**Issue by harrisj** [natehole/TroikaDiscordBot#36]
It seems like [Technical Grimoire's Troika background generator](https://www.technicalgrimoire.com/troikagenerator) has become a defacto standard, supported by multiple creators. So, let's switch to supporting that instead and work on defining other similar formats for spells and weapons

**Comment by natehole** [comment]
What do have in mind? Is the plan to get permission and use the backgrounds from Technical Grimoire's Troika for character creation via the discord bot?

**Comment by harrisj** [comment]
Yeah, potentially. But it also seems to have gotten traction with multiple creators. Theoretically, the bot could fetch on startup if that were a concern
